go/vt/sqlparser/ast.go

```diff
@@ -2707,6 +2707,34 @@ type (
 	}
 
 	CountStar struct {
+		Name string
```

Comment:
@GuptaManan100 would this reduce the cost of this change? Enough to matter?

```
type CountStarName int

var (
        CountStarLower CountStarName iota
        CountStarCapital
        CountStarUpper
)
```
_(`count`, `Count` and `COUNT` only)_

`Name string` would become `Name CountStarName` in `CountStar struct`

Reply:
It could matter. It would definitely reduce the size of the struct, but how meaningful that overall change is going to be to runtime, we'll have to test. I don't think it would be very significant either way, because we do need to print the string eventually during formatting.